base/applications/rapps/include/misc.h

```diff
@@ -6,6 +6,17 @@
 #define EPOCH_DIFF 116444736000000000 //FILETIME starts from 1601-01-01 UTC, UnixTime starts from 1970-01-01
 #define RATE_DIFF 10000000
 
+#ifdef _M_IX86
+#define CurrentArchitecture L"x86"
+#elif defined(_M_AMD64)
+#define CurrentArchitecture L"amd64"
+#elif defined(_M_IA64)
+#define CurrentArchitecture L"ia64"
+#elif defined(_M_ARM)
+#define CurrentArchitecture L"arm"
```

Comment:
```suggestion
#elif defined(_M_ARM)
#define CurrentArchitecture L"arm"
#elif defined(_M_ARM64)
#define CurrentArchitecture L"arm64"
```

Reply:
`_M_ARM` and `_M_ARM64` are mutually exclusive?

Reply:
Absolutely.